speech/apiv1/adaptation_client_create_custom_class_basic_example_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 Google LLC
```

Comment:
Why are there two identical files with different paths/name? Is this just testing out different locations?

The only thing I would point out is that it would be preferable to have and indication of the snippet being generated either on the path or the file name, or both and I can't see this on this file here.

Reply:
Oh, replying myself, is this because of @tbpg 's https://github.com/googleapis/google-cloud-go/pull/6688#issuecomment-1274743227? Then that's totally fine, SnippetGen won't be opinionated about language specific concerns.

Reply:
gapic-generator-go produces `speech/apiv1/adaptation_client_create_custom_class_basic_example_test.go`, which is intended for pkg.go.dev. Post-processing copies the content to `internal/generated/snippets/speech/apiv1/AdaptationClient/CreateCustomClassBasic/main.go` and adds region tags.